db/migrate/20250628172500_create_notifications.rb

```diff
@@ -0,0 +1,14 @@
+class CreateNotifications < ActiveRecord::Migration[8.0]
+  def change
+    create_table :notifications, id: {type: :bigint, unsigned: true} do |t|
+      # We shouldn't need an index on the user_id alone because the unique index below provides user_id in the leftmost column
+      t.references :user, null: false, foreign_key: true, type: :bigint, unsigned: true, index: false
+      t.references :notifiable, polymorphic: true, null: false, type: :bigint, unsigned: true
```

Comment:
It's future-proofing, but please add a `t.string :token, null: false` and in the model `include Token`.

Reply:
added